packages/wallets/src/solana/wallet.ts

```diff
@@ -85,11 +89,12 @@ export class SolanaSmartWallet extends SolanaWallet {
         apiClient: ApiClient,
         publicKey: PublicKey,
         adminSignerInput: SolanaSignerInput,
+        callback: Callbacks = {},
         client: { public: Connection } = {
             public: getConnectionFromEnvironment(apiClient.environment),
         }
     ) {
-        super(apiClient, publicKey, client);
+        super(apiClient, publicKey, callback, client);
```

Comment:
I feel like callbacks should be the last param

Reply:
makes sense https://github.com/Crossmint/crossmint-sdk/pull/985/commits/856a9eb2d24641276899382f9d0c392e88f28fcc